.github/workflows/sdk_validation.yaml

```diff
@@ -19,8 +19,6 @@ jobs:
     timeout-minutes: 10
     env:
       SHIPPO_TOKEN: ${{ secrets.SHIPPO_TOKEN }}
-    concurrency:
```

Comment:
what was this used for before? how did it get here?

Reply:
My assumption? To prevent concurrent runs with the different python versions.
I think you might be able be able to [answer that](https://github.com/goshippo/shippo-python-sdk/commit/4556d95e2b8e8741ad1264dccc52db0ad56fb06d) better than me lol

Reply:
Hmm ok well as long as it's not breaking this way seems fine to me